shell/packages/sandstorm-accounts-ui/login_buttons.html

```diff
@@ -142,9 +142,12 @@
       <li><button type="button" class="login-dev-account" data-name="Dave Dev User">
           Dave
       </button></li>
-      <li><button type="button" class="login-dev-account" data-name="Eve Dev User">
-          Eve
-      </button></li>
+      <li>
+        <div class="flex-box">
+          <input name="name" class="custom-name" type="text" placeholder="Eve" required>
```

Comment:
Doesn't marking this required mean that you can't submit the form by clicking the other buttons, e.g. "Bob"?